src/schema/objects/columns.yaml

```diff
@@ -794,6 +794,8 @@ volume_type:
     - $ref: objects.enums.m0scan.value
     - $ref: objects.enums.deltam.value
     - $ref: objects.enums.cbf.value
+    - $ref: objects.enums.noRF.value
+    - n/a
```

Comment:
I think in general we don't require columns to declare `n/a` as valid, since it's always valid. It would be good to test the schema validator with an example:

```console
$ python -m venv .venv
$ source .venv/bin/activate
$ pip install -e tools/schemacode
$ bst export > src/schema.json
$ BIDS_SCHEMA=$PWD/src/schema.json deno run -A https://github.com/bids-standard/bids-validator/raw/deno-build/bids-validator.js $DATASET
```

Reply:
I tested using the current version of the deno validator and it passes if n/a is in the aslcontext file, so I guess I don't need to include it as an explicit enum.